src/main/java/htsjdk/variant/vcf/AbstractVCFCodec.java

```diff
@@ -222,6 +228,7 @@ protected VCFHeader parseHeaderFromLines( final List<String> headerStrings, fina
             }
         }
 
+        if (!contigsExist) LOG.warn("There are no contig lines in vcf file header");
```

Comment:
I don't think we need to warn here; as with SAM/BAM, as far as I can tell contig lines are not required in a VCF. So I think we can revert these changes.